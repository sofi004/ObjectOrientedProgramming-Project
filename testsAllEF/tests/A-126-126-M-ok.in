5
22
2
A
2
U
0
0
