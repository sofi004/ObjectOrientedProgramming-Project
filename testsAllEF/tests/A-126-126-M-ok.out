Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Opção inválida!
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Função a procurar: 2;3|48=ADD(2;2,5)
4;2|34=AVERAGE(2;1:2;2)
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Função a procurar: 3;3|9=MUL(4;1,4;1)
1;3|-1=SUB(2,3)
4;3|-3=SUB(2,5)
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 